File: src/main/java/pl/dmichalski/algorithms/_9_sorting/selection/Runner.java
package pl.dmichalski.algorithms._9_sorting.selection;

import java.util.Arrays;

/**
 * Write a function called sort, that accepts integer array
 * and returns sorted array using selection sort algorithm.
 */
public class Runner {

    public static void main(String[] args) {
        int[] values = {3, 2, 5, 1, 4};

        SelectionSortService sortService = new SelectionSortService();
        int[] result = sortService.sort(values);
        System.out.println("result = " + Arrays.toString(values));
    }

}

File: src/main/kotlin/pl/dmichalski/algorithms/_9_sorting/quick/Runner.kt
package pl.dmichalski.algorithms._9_sorting.quick

import pl.dmichalski.algorithms._9_sorting.SortServiceRunner

/**
 * Write a function called sort, that accepts integer array
 * and returns sorted array using quick sort algorithm.
 */
object Runner {

    @JvmStatic
    fun main(args: Array<String>) {
        val sortService = QuickSortService()
        SortServiceRunner.runTestCase(sortService)
    }

}
